account_invoice_report_payment_info/models/account_invoice.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.move"
+
+    def _get_reconciled_info_JSON_values(self):
+        res = super()._get_reconciled_info_JSON_values()
```

Comment:
This method does not exist anymore. So this is no having effect. 

Check where is the data collected on the report to replicate the behaviour.